2.Installation/2.3.InstallUsingComposer.md

```diff
@@ -0,0 +1,16 @@
+2.1 Installing using Composer
+=============================
+ 
+Installing Fluid Powered TYPO3 with Composer is a simple command-line operation:
+
+Prerequisites: 
+- You have to use TYPO3 in Composer-mode. Please look at https://composer.typo3.org/ for more information. 
+- It is recommended that you have added `https://composer.typo3.org/` to your list of repositories.
+
+1. Open a command-line tool and navigate to the folder where the composer.json for your TYPO3-installation exists.

```

Comment:
It is recommended not to use the root composer JSON from TYPO3 but create a fresh one. This advise is then correct except for the part about needing to browse to the folder in question - instead, this should be done from the *project root folder* (which may or may not have been created based on the typo3_src and may or may not contain an existing composer.json; regardless how that source was created the require command works)